index.js

```diff
@@ -0,0 +1,188 @@
+'use strict'
+// Copyright (c) 2018 inspiredware
+
+var path = require('path')
+var pkg = require(path.resolve('package.json'))
+
+var versionArray = process.version
+  .substr(1)
+  .replace(/-.*$/, '')
+  .split('.')
+  .map(function (item) {
+    return +item
+  })
+
+/**
+ *
+ * A set of utilities to assist developers of tools building
+ * [N-API](https://nodejs.org/api/n-api.html#n_api_n_api) native add-ons.
+ *
+ * The main repository can be found [here](https://github.com/inspiredware/napi-build-utils).
+
+ * @module napi-build-utils
+ */
+
+/**
+ * Implements a consistent name of `napi` for N-API runtimes.
+ *
+ * @poram {string} runtime The runtime string.
+ * @returns {boolean}
+ */
+exports.isNapiRuntime = function (runtime) {
+  return runtime === 'napi'
+}
+
+/**
+ * Determines whether the specified N-API version is supported
+ * by both the currently running Node instance and the package.
+ *
+ * @poram {string} napiVerison The N-API version to check.
+ * @returns {boolean}
+ */
+exports.isSupportedVersion = function (napiVerison) {
+  var version = parseInt(napiVerison, 10)
+  return version <= exports.getNapiVersion() && exports.packageSupportsVersion(version)
+}
+
+/**
+ * Determines whether the specified N-API version is supported by the package.
+ * The N-API version must be preseent in the `package.json`
+ * `binary.napi_versions` array.
+ *
+ * @poram {number} napiVerison The N-API version to check.
+ * @returns {boolean}
+ * @private
+ */
+exports.packageSupportsVersion = function (napiVerison) {
+  if (pkg.binary && pkg.binary.napi_versions) {
```

Comment:
Any reason to check `isArray(napi_versions)` or something like that, just in case? That may be overkill but just a thought

Reply:
👍🏼